snafu/run_snafu.py

```diff
@@ -87,8 +87,9 @@ def main():
                                                  ssl_context=ssl_ctx, use_ssl=True)
             else:
                 es = elasticsearch.Elasticsearch([_es_connection_string], send_get_body_as='POST')
-            logger.info("Connected to the elasticsearch cluster with info as follows:{0}".format(
-                str(es.info())))
+            logger.info("Connected to the elasticsearch cluster with info as follows:")
+            logger.info(json.dumps(es.info(), indent=4))
```

Comment:
This has the potential to be large, you might want to trim this to a max length to avoid logging issues with large messages.

Reply:
Ack, will create an issue and address in separate PR.